bitmap-plasma/app/build.gradle

```diff
@@ -19,7 +19,7 @@ model {
 
     android.buildTypes {
         release {
-            isMinifyEnabled = false
+            minifyEnabled = false
```

Comment:
it seems that this is optional though, even without it I'm able to hit a breakpoint.